src/javascript/JContent/ContentRoute/ContentBreadcrumb/ContentPath/ContentPath.jsx

```diff
@@ -20,10 +20,12 @@ const renderItems = (items, onItemClick) => {
 
 const ContentPath = ({items, onItemClick}) => {
     return (items.length > 0) &&
-        <Breadcrumb>
-            {renderItems(items, onItemClick)}
-            <></>
-        </Breadcrumb>;
+        <>
+            <Breadcrumb>
+                {renderItems(items, onItemClick)}
+            </Breadcrumb>
+            <Separator variant="vertical"/>
```

Comment:
It should be moved ContentBreadcumb.jsx instead (between ContentPath and ContentType components).

Reply:
I don't understand? What do you mean?

Reply:
The "Separator" component should be moved out of ContentPath. It does not make sense here since we are no longer leveraging the one from the Breadcrumb. Proper location should be one level above - between <ContentPath/> and <ContentType/> components (ie. in ContentBreadcrumb.jsx).

Reply:
here: https://github.com/Jahia/jcontent/blob/master/src/javascript/JContent/ContentRoute/ContentBreadcrumb/ContentBreadcrumb.jsx

Reply:
I finally see your point of view, from the way I put it here, to avoid in case we don't have a content path (items.length === 0) so we'll have an ugly pipe (separator) and a content type; this particular case can happen if we are at the root level of the content ex: /contents or /home

Reply:
Fair point. Thanks for clarifying!

Reply:
you're welcome